packages/clarity-js/src/data/encode.ts

```diff
@@ -110,7 +110,7 @@ export default function(event: Event): void {
             let extractKeys = extract.keys;
             for (let e of extractKeys) {
                 tokens.push(e);
-                tokens.push(extract.data[e]);
+                tokens.push([].concat(...extract.data[e]));
```

Comment:
All you are doing here is making a shallow copy of the array into the tokens. Is that your intent?

Reply:
we are converting the array of array into a single array